Specs/Number/Hindi/NumberRangeModel.json

```diff
@@ -527,8 +498,7 @@
   },
   {
     "Input": "x 10 से बड़ी और 20 से छोटी है. y 50 से अधिक नहीं है और 20 से कम नहीं है.",
-    "NotSupported": "javascript, python",
-    "NotSupported": "dotnet",
+    "NotSupported": "javascript, python, dotnet",
```

Comment:
Cases like this one are the ones that should have "Comment" to document the reason for the temporary lack of support.
Also, javascript and python are already listed in NotSupportedByDesign, so they should not be tagged as NotSupported too.

Reply:
added comments, removed "NotSupported": "javascript, python",